analyzers/SonarAnalyzer.Shared/SonarAnalyzer.Shared.projitems

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MSBuildAllProjects Condition="'$(MSBuildVersion)' == '' Or '$(MSBuildVersion)' &lt; '16.0'">$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <SharedGUID>892cf3fa-3be2-42e8-a7e6-76ae72864dec</SharedGUID>
+  </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <Import_RootNamespace>SonarAnalyzer.Shared</Import_RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)Extensions\SyntaxTokenExtensions.cs" />

```

Comment:
This list [tends to grow](https://github.com/SonarSource/sonar-security/blob/master/its/projects/internal/InternalCSharp/Shared.All/Shared.All.projitems) and therefore needs to be maintained.

Can we use some `*` trick here like [this](https://github.com/SonarSource/sonar-dotnet/blob/c0a120782f45321265c499f668c5f2b3ee885364/analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj#L50-L54)?

Something like
```suggestion
    <Compile Include="$(MSBuildThisFileDirectory)**\*" />
```

Reply:
This has the side effect of adding additional files like `SonarAnalyzer.Shared.projitems`. I've updated the pattern to `**\*.cs`